owscheck.py

```diff
@@ -195,3 +196,13 @@ def get_service(self):
 
     def get_layer_names(self):
         return self._layer_names
+
+    def _reduced_bbox(self, bbox):
+        xmin, ymin, xmax, ymax = bbox
+        p = [xmin+0.49*(xmax-xmin),
+             ymin+0.49*(ymax-ymin),
+             xmax-0.49*(xmax-xmin),
+             ymax-0.49*(ymax-ymin)]
```

Comment:
0.49 does not mean reducing but 49% If we go over 50% then "xmin" become higher than "xmax".

To demonstrate "visually" copy this code in the JS pan of this [code pen](https://codepen.io/mina-tokyo/pen/dypKemy) (remove every JS).

```js
const zoom = 7;
const source = ''

const map = L.map('example-map');
L.tileLayer('https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', {
        maxZoom: 19,
        attribution: '&copy; <a href="https://openstreetmap.org/copyright">OpenStreetMap contributors</a>'
      }).addTo(map);
map.setView([45,5], zoom);


var originalBound = [[2.018463747207982, 44.597830110962].reverse(), [4.546589166293389, 46.815108752174545].reverse()];

var tenpercentBound = [[3.2572452025598313, 45.684296645156145].reverse(), [3.3078077109415394, 45.7286422179804].reverse()]

// create an orange rectangle
L.rectangle(originalBound, {color: "#ff7800", weight: 1}).addTo(map);
L.rectangle(tenpercentBound, {color: "#fffff", weight: 10}).addTo(map);

// zoom the map to the rectangle bounds
map.fitBounds(originalBound);
```

Reply:
> 0.49 does not mean reducing but 49% If we go over 50% then "xmin" become higher than "xmax".

Sure. Stupid me ;-)

Reply:
I had to take a sheet of paper and draw it to be sure ;-) not practicing (basic) math makes me very rusty  !